katka/models.py

```diff
@@ -8,8 +8,19 @@
 
 class Team(AuditedModel):
     public_identifier = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    slug = models.SlugField(max_length=10)
```

Comment:
is this unique?

Reply:
hm, wanted to check this, but forgot. I just checked and it's not. I'll fix this.

Reply:
I updated the code, how do you feel about it?